ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -250,35 +250,25 @@ objects:
       - name: artifacts
         mountPath: /tmp/artifacts
       env:
-      - name: TYPE
-        value: ${CLUSTER_TYPE}
```

Comment:
I expect we need to keep this [documented variable][1] for our future GCP, etc. support.  CC @smarterclayton, @stevekuznetsov

[1]: https://github.com/openshift/release/blame/2ee443911599c3f20bf6a4f4f744b4a9917dee8b/ci-operator/README.md#L328